cadquery/FCAD_script_generator/GW_QFP_SOIC_SSOP_TSSOP_SOT/cq_parameters_tssop.py

```diff
@@ -0,0 +1,1545 @@
+# -*- coding: utf8 -*-
+#!/usr/bin/python
+#
+# This is derived from a cadquery script for generating QFP/GullWings models in X3D format.
+#
+# from https://bitbucket.org/hyOzd/freecad-macros
+# author hyOzd
+#
+# Dimensions are from Jedec MS-026D document.
```

Comment:
Should be MO-153G

Reply:
Thanks for pointing it out. Corrected in #339